workers/webpack.js

```diff
@@ -119,10 +121,32 @@ function webpack(opts, callback) {
   }, function (err, stdout, stderr) {
     if (err) {
       err.output = stdout + stderr;
+      //
+      // Rebuild and rerun if the error is special
+      //
+      if (err.message.includes('npm rebuild')) {
```

Comment:
Doesn't the usage of `includes` limit the the node version this can be used on as it's ES6? Wouldn't a simple ES6 indexOf check be more suitable here? Not saying it's wrong, just wondering what our node version requirements are.